pkg/azuredisk/nodeserver.go

```diff
@@ -489,3 +510,111 @@ func (d *Driver) ensureBlockTargetFile(target string) error {
 
 	return nil
 }
+
+func (d *Driver) nodePublishEphemeralVolume(ctx context.Context, req *csi.NodePublishVolumeRequest) (*csi.NodePublishVolumeResponse, error) {
+	volumeId := req.GetVolumeId()
+	ephemeralVolumeId := fmt.Sprintf(ephemeralVolumeIdTemplate, volumeId)
+
+	capacity := req.GetVolumeContext()["capacity"]
+	size, err := strconv.Atoi(strings.TrimSuffix(capacity, "Gi"))
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Unable to parse capacity %v", err)
+	}
+	if size <= 0 {
+		return nil, status.Errorf(codes.Internal, "size(%v) is illegal, please provide the positive size value", size)
+	}
+
+	parameters := make(map[string]string)
+	parameters["skuname"] = req.GetVolumeContext()["storageAccountType"]
```

Comment:
could you just pass all VolumeContext as parameter since there could be other parameters defined in VolumeContext and it's case insensitive: 
https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/e1c2613898996dbaf8223c866c791a14335598c2/pkg/azuredisk/controllerserver.go#L105-L113

Reply:
done